docs/kb/install-veba.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: docs
+toc_id: install-veba
+title: VMware Event Broker Appliance
+description: Deploying VMware Event Broker Appliance
+permalink: /kb/install-veba
+cta:
+ title: Deploy a Function
+ description: At this point, you have successfully deployed the VMware Event Broker Appliance and you are ready to start deploying your functions!
+ actions:
+  - text: Check the [Knative Echo Function](https://github.com/vmware-samples/vcenter-event-broker-appliance/tree/master/examples/knative/powershell/kn-ps-echo){:target="_blank"} to quickly get started
+---
+
+# Deploy VMware Event Broker Appliance
+
+Customers looking to seamlessly extend their vCenter by either deploying our prebuilt functions or writing your own functions can get started quickly by deploying VMware Event Broker Appliance with Knative as the Event Processor
+
+<!-- omit in toc -->
+## Table of Contents
+
+- [Deploy VMware Event Broker Appliance](#deploy-vmware-event-broker-appliance)
+  - [Requirements](#requirements)
+  - [Step 1 - Download OVA](#step-1---download-ova)
+  - [Step 2 - Deploy OVA](#step-2---deploy-ova)
+    - [Networking (Required)](#networking-required)
+    - [Proxy Settings (Optional)](#proxy-settings-optional)
+    - [OS Credentials (Required)](#os-credentials-required)
+    - [vSphere (Required)](#vsphere-required)
+    - [Horizon (Optional)](#horizon-optional)
+    - [Webhook (Optional)](#webhook-optional)
+    - [Custom TLS Certificate Configuration (Optional)](#custom-tls-certificate-configuration-optional)
+    - [Syslog Server Configuration (Optional)](#syslog-server-configuration-optional)
+    - [zAdvanced (Optional)](#zadvanced-optional)
+  - [Step 3 - Verification](#step-3---verification)
+
+## Requirements
+
+- 6 vCPU and 8GB of memory for VMware Event Broker Appliance
+- vCenter Server 7.x or greater
+  - **The VEBA UI requires vCenter Server 7.0 or greater**
+- vCenter TCP/443 accessible from Appliance IP address
+- Account to login to vCenter Server (readOnly is sufficient)
+
+## Step 1 - Download OVA
+
+Download the VMware Event Broker Appliance (OVA) from the [VMware Fling site](https://flings.vmware.com/vmware-event-broker-appliance){:target="_blank"}.
```

Comment:
Need to update this to via.vmware.com/flings (since we don't have a specific landing URL for VEBA Fling)

Reply:
via.vmware.com/flings does not work but vmwa.re/flings does 😉